CMakeLists.txt

```diff
@@ -198,12 +198,20 @@ if(cxxmodules)
   endif()
 endif(cxxmodules)
 
+if(MSVC)
+  set(_os_cat "type")
+else()
+  set(_os_cat "cat")
+endif()
+file(TO_NATIVE_PATH "${CMAKE_BINARY_DIR}/include/module.modulemap.extra" _from_native)
+file(TO_NATIVE_PATH "${CMAKE_BINARY_DIR}/include/module.modulemap" _to_native)
+
 add_custom_target(copymodulemap DEPENDS "${CMAKE_BINARY_DIR}/include/module.modulemap")
 add_custom_command(
                   OUTPUT "${CMAKE_BINARY_DIR}/include/module.modulemap"
                   DEPENDS build/unix/module.modulemap "${CMAKE_BINARY_DIR}/include/module.modulemap.extra"
                   COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_SOURCE_DIR}/build/unix/module.modulemap" "${CMAKE_BINARY_DIR}/include/module.modulemap"
-                  COMMAND cat "${CMAKE_BINARY_DIR}/include/module.modulemap.extra" >> "${CMAKE_BINARY_DIR}/include/module.modulemap"
+                  COMMAND ${_os_cat} "${_from_native}" >> "${_to_native}"
```

Comment:
Hi @bellenot, here it would be nicer if we used CMake commands instead of cat or type. CMake has `file(READ ...)` and `file(APPEND ...)` which can do the job in a portable way.

Reply:
https://stackoverflow.com/questions/15282444/using-cmake-how-can-i-concat-files-and-install-them

Reply:
Thanks Guilherme! I will try

Reply:
For the record: I merge it the way it is now, but I agree this should be reviewed at some point and converted with some portable CMake commands